gateway/src/main/java/org/georchestra/gateway/app/GeorchestraGatewayApplication.java

```diff
@@ -59,6 +61,7 @@
 @Slf4j
 @SpringBootApplication
 @EnableConfigurationProperties(LdapConfigProperties.class)
+@EnableAutoConfiguration(exclude = RabbitAutoConfiguration.class)
```

Comment:
I am not a spring boot expert, but I find it strange to annotate the main spring-boot application class with `EnableAutoConfiguration` ; looking at the spring boot sources, it seems to be mainly used onto `*AutoConfiguration` classes.

Also assuming that the exclude parameter of the annotation will do the same as the `spring.autoconfigure.exclude` yaml list, what does this PR bring exactly ? It sounds to me that this change is equivalent to the previous state of the code

Reply:
Yes, seems the same, But on Mel using this patch works, with property "spring.autoconfigure.exclude"it does not work.
This is required to make gateway works whenever rabbitmq is off on Mel.

Reply:
@groldan, need your input please .

Reply:
Maybe the exclude in application.yml doesn't work because in the Mel deployment it's been overridden with one that doesn't have the exclude statement? Like in -Dspring.config.location=...

Reply:
> Maybe the exclude in application.yml doesn't work because in the Mel deployment it's been overridden with one that doesn't have the exclude statement?

indeed, it looks like it is the case here